oeis.py

```diff
@@ -139,6 +138,22 @@ def A000045(start: int = 0, limit: int = 20) -> Collection[int]:
         sequence.append(sequence[i - 1] + sequence[i - 2])
     return sequence
 
+@oeis
+def A000566(start: int = 0, limit: int = 20) -> Collection[int]:
```

Comment:
Your function accepts a `start` value, you should use it.

Reply:
Ah oui !